docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
 
-. In the dashboard of your space, select the *Plan* tab at the top of the page to view the default *Backlog* view of the planner.
-. Select the *Show Tree* check box to see the work items and their children in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
-. Use the quick-add option to create a *Scenario* as follows:
-.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
-+
-image::quickadd_scenario.png[Quick add scenario]
-+
-.. In the *Work Item Title* field, add *Enable an end to end, developer work-flow, using the continuous integration and deployment framework* as the title.
-.. Press kbd:[Enter] to create the scenario.
+. In the dashboard of your space, select the *Plan* tab at the top of the page. The default *Backlog* view displays the *Work Items* for your space.
+. Select the *Show Tree* check box to use the quick-add and in-line add options to quickly create workitems and their children.
+. In the *Add Work Item* pane, select *Theme* from the drop-down list, type *Enable a complete developer work-flow in a continuous integration and delivery framework* as the title, and press kbd:[Enter] to create the theme.
 
-. Create a child *Experience* for the *Scenario* using the in-line add option as follows:
-.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
+. Create a child *Epic* for the *Theme* as follows:
+.. In the work item list, click the (+) icon adjacent to the theme to see the in-line *Add Child Work Item* pane below it.
 +
 image::gs_inline-add_eg.png[Inline Add]
 +
+.. Select *Epic* from the drop-down list, type the title *User should be able to launch applications through a simple work-flow*, and Press kbd:[Enter].
 
-.. Select *Experience* from the drop-down list, add the title *User should be able to launch applications through a simple work-flow*, and Press kbd:[Enter].
-
-. Create an execution oriented child *Feature* for the *Experience* using the in-line add option with the title *User should be able to easily launch a sample Vert.x application using a smooth developer work-flow*.
-
-. Create the following child *Task* work items to implement the *Feature* using the in-line add option:
-
-* <<creating_new_project-hello-world,*Launch a sample quickstart application*>>
-* <<approving_build_pipeline,*Test the application in Stage and approve to the Run environment*>>
-* <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
-* <<reviewing_publishing_changes-hello-world,*Review and publish changes to your codebase*>>
+. Using the in-line add option create an execution oriented child *Story* for the *Epic* with the title *User should be able to easily launch a sample Vert.x application* and a child *Task* for the story titled *Review and publish changes to your codebase*.
```

Comment:
This is IMO very confusing. If it's necessary to create another work item, we should have specific steps for this part. I clicked what I assume is the in-line option but it has both Story and Epic types in the drop down menu and it's not clear which one to select or why, the title should be shorter and to the point. This reads like several steps mashed together at the moment.

Reply:
Yes I clubbed 2 separate steps here since they were to be done in a similar manner as highlighted with screenshots in the previous step. The other reason was to cut down on the number of steps. The inline option does not have both options, please see the screenshot for the inline add. It was precisely for this reason I retained substeps for inline add as well as the screenshot as I saw potential for confusion. Have you tried rendering this doc and then reviewed it. I hope along with the screenshot this is easier to follow.